unity/Assets/Maroon/SharedEntities/Player/ModeFirstPerson/ModeFirstPerson.cs

```diff
@@ -1,14 +1,16 @@
-﻿using UnityEngine;
+﻿using System.Collections;
+using Maroon.GlobalEntities;
+using UnityEngine;
 using UnityEngine.Events;
 
-[RequireComponent(typeof(CharacterController), typeof(ModeFirstPersonInputHandler), typeof(AudioSource))]
+[RequireComponent(typeof(CharacterController), typeof(ModeFirstPersonInputHandler))]
 public class ModeFirstPerson : MonoBehaviour
 {
     [Header("References")]
     [Tooltip("Reference to the main camera used for the player")]
     public Camera playerCamera;
     [Tooltip("Audio source for footsteps, jump, etc...")]
-    public AudioSource audioSource;
+    private AudioSource audioSource;
```

Comment:
Set to private, since it's not accessed by Inspector anymore and only used inside this class.